lib/windows/pagination.py

```diff
@@ -0,0 +1,201 @@
+# coding=utf-8
+import kodigui
+from lib import util
+
+
+class MCLPaginator(object):
+    """
+    A paginator for ManagedControlList instances
+    """
+    control = None
+    pageSize = 8
```

Comment:
I think it would be worthwhile to set this to something like 26, so that most seasons don't need paging at all - or at the very least have the initial load be 26 and then page at a lower count.